keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -309,6 +632,15 @@ At beta we will revisit if e2e buckets are warranted for e2e node, and after gat
 
 - Feature implemented behind a feature flag - KubeletEnsureSecretPulledImages
 - Initial e2e tests completed and enabled - No additional e2e identified as yet
+- The kubelet cache is implemented solely via files. The implementation leverages
+  interfaces in order to abstract the cache storage (in memory/via files/file-based database/...)
+  and the storage format (JSON/CBOR/...). This is to allow us to later experiment with
+  these various aspects to achieve optimal performance for a picked data transparency model.
+
+#### Beta
+
+- Kubelet cache is duplicated in memory and on disk. This is more resilient to I/O
+  operation errors when reading from disk.
```

Comment:
IIUC,  for FG `KubeletEnsureSecretPulledImages`
- alpha: 
  - FG default false, imagePullCredentialsVerificationPolicy doesn't work. Behavior is same with `NeverVerify`.
  - When enabled, the imagePullCredentialsVerificationPolicy default is NeverVerifyPreloadedImages.
- beta: 
  - FG default true, imagePullCredentialsVerificationPolicy default to NeverVerifyPreloadedImages?
  - If you want to use the old behavior, change imagePullCredentialsVerificationPolicy=NeverVerify. Or you can just disable the FG. Either is OK.

There would be a behavior change when promoting the FG to beta.